composer.json

```diff
@@ -8,12 +8,13 @@
             "email": "stephen@stecman.co.nz"
         }
     ],
+    "minimum-stability": "beta",
     "require": {
         "php": ">=5.3.2",
-        "symfony/console": "~2.3 || ~3.0"
+        "symfony/console": "~2.3 || ~3.0 || ~4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.4"
+        "phpunit/phpunit": "~4.4 || ~5.7"
```

Comment:
Let's add PHPUnit 6.x here as well.

Reply:
Added, but note that 4.x & 6.x are incompatible and 6.x not supports \PHPUnit_Framework_TestCase, you should use \PHPUnit\Framework\TestCase at least in your tests. 5.7 supports both FQCN cases

Reply:
Ah, then please (if possible) send a separate PR where you replaces `\PHPUnit_Framework_TestCase` with `TestCase` + `use PHPUnit\Framework\TestCase;`.